scripts/get_usernames.py

```diff
@@ -0,0 +1,20 @@
+import csv
```

Comment:
Fix the style issues here - 4 space indents and snake case for variable names.

Also suggest calling this convert_usernames.py since it doesn't really 'get' anything.